### ISSUE TITLE
Feature: Allow to disable HTTPS 

### ISSUE THREAD

**Issue by xeor** [Galexrt/docker-zulip#57]
I need to make a couple of changes that is not part of the main django configuration.
I was very glad to see the `post-setup.d` part in the entrypoint (https://github.com/Galexrt/docker-zulip/blob/master/entrypoint.sh#L474). But looks like the check there is wrong.
See;
```
    if [ "$(ls -A "$DATA_DIR/post-setup.d/")" ]; then
        echo "No post setup scripts found in \"$DATA_DIR/post-setup.d/\"."
        return 0
    fi
```

It is missing a `not`. And those 4 lines are not even necessary, since the loop below wont fire any script if it can't find any files.. Maybe these 4 lines should just be removed?

**Comment by Galexrt** [comment]
I'm working on a fix.

Could you tell me what changes you want to make to the Django config? Because then I could consider adding environment variables for the configuration of Django.

Thanks for the bug report!

**Comment by xeor** [comment]
Thansk for the quick response! :)

I am using a proxy in front of this container that takes care of both https and some other things. So I had to change out the nginx config, killing https, and killing the redirect.

**Comment by Galexrt** [comment]
@xeor I could add an environment Variable for disabling https and https redirect from the nginx in the container. So you don't need to modify the container files. 
Would that be okay for you?

**Comment by xeor** [comment]
That would be a very acceptable solution! I would imagine many people wanting to do this as well, so a generic fix would be good.

The `post-setup.d` issue would still be nice to have tho. Since it's undocumented, maybe you can write a little comment in the `entrypoint` script asking people to submit an issue, if their `post-setup` script fixes something that should be implemented as an environment-variable..

Thanks!

**Comment by Galexrt** [comment]
I'm not removing the `post-setup.d` feature.
I just add an environment Variable like `DISABLE_HTTPS`, to disable nginx https and the https redirect.

**Comment by xeor** [comment]
Cool.. But the `post-setup.d` feature is still broken, as it says that it's not able to find any files, if it finds files..

**Comment by Galexrt** [comment]
As I already said, I'm going to fix the `post-setup.d ` feature.
I can push the fix in about 30 minutes, after the image has been rebuilt, you can repull the image (`docker pull quay.io/galexrt/zulip:latest`).

**Comment by Galexrt** [comment]
Can you try again with version `v1.3.9-2` of the image?
In the `v1.3.9-2` I fixed the `post-setup.d` script execution.

**Comment by Galexrt** [comment]
The `DISABLE_HTTPS` env variable has been implemented in the `dev-v1.3.10` branch.
Please don't use the `dev-v1.3.10` image.
Closing as the feature has been implemented and we'll just have to wait until Zulip merges the official Dockerfile Pull request for this to work.
